pupil_src/shared_modules/calibration_routines/gaze_mappers.py

```diff
@@ -236,7 +236,7 @@ def deinit_ui(self):
 
     def _map_binocular(self, p0, p1):
         if self.multivariate:
-            gaze_point = self.map_fn(p0["norm_pos"], p1["norm_pos"])
+            gaze_point = self.map_fn([*p0["norm_pos"]] + [*p1["norm_pos"]])
```

Comment:
Please reevaluate if change is necessary. If yes, replace by `list(pX["norm_pos"])`